guestmetric/guestmetric_linux.go

```diff
@@ -182,7 +182,7 @@ func (c *Collector) CollectNetworkAddr() (GuestMetric, error) {
 	current := make(GuestMetric, 0)
 
 	var paths []string
-	vifNamePrefixList := [...]string{"eth", "eno", "ens", "emp", "enx"}
+	vifNamePrefixList := [...]string{"eth", "eno", "ens", "emp", "enx", "enX"}
```

Comment:
What is the difference between 'enx' and 'enX'? Only ArchLinux vm installed on Xcp-ng will named enX?

Reply:
the point is that you loop over 

```
 		prefixPaths, err := filepath.Glob(fmt.Sprintf("/sys/class/net/%s*", prefix))
```

and it's case sensitive, that's how you get the IP info.

On archlinux, it does not find because enx != enX.

Reply:
Also not sure if it's only Arch, I need to try from other OS....